tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -87,7 +87,9 @@ def run_pipeline_test(self, speech_recognizer, examples):
         if speech_recognizer.type == "ctc":
```

Comment:
With the addition of `type == "seq2seq_whisper"` we can do `speech_recognizer.type in {"ctc", "seq2seq_whisper"}` I think